netlink-packet-route/src/rtnl/tc/nlas/filter/mod.rs

```diff
@@ -0,0 +1,3 @@
+// SPDX-License-Identifier: MIT
+
+pub mod u32;
```

Comment:
I got confused for a second :laughing:

Reply:
The naming is really confusing. I tried naming the u32.rs file as U32.rs so that I could use the U32 mod. but there is a compile warning.

Reply:
Yeah this is fine :)